lang/test/org/partiql/lang/syntax/SqlParserTest.kt

```diff
@@ -15,9 +15,8 @@
 package org.partiql.lang.syntax
 
 import com.amazon.ion.Decimal
-import com.amazon.ionelement.api.ionDecimal
-import com.amazon.ionelement.api.ionInt
-import com.amazon.ionelement.api.ionString
+import com.amazon.ion.IonInt
+import com.amazon.ionelement.api.*
```

Comment:
nit: as per our [style guide](https://github.com/partiql/partiql-lang-kotlin/blob/main/docs/dev/CODE-STYLE.md#imports), try not to use star imports